src/test/java/pages/HomePage.java

```diff
@@ -40,6 +48,39 @@ public void createPlaylist(String name) throws InterruptedException {
         new WebDriverWait(driver, Duration.ofSeconds(1)).until(ExpectedConditions.invisibilityOfElementLocated(By.cssSelector("[name='name']")));
```

Comment:
**Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )** 
![image](https://user-images.githubusercontent.com/104761482/204586391-be5acfac-1f76-476e-a5d1-14231b553cce.png)

For example
`wait.until(ExpectedConditions.invisibilityOf(playlistNameField));`